### ISSUE TITLE
Readme show a xml report as an example for a ruleset

### ISSUE THREAD

**Issue by DannyvdSluijs** [phpmd/phpmd#530]
From the read the following example is available as a ruleset:
```xml
<?xml version="1.0" encoding="UTF-8" ?>
<pmd version="0.0.1" timestamp="2009-12-19T22:17:18+01:00">
  <file name="/projects/pdepend/PHP/Depend/DbusUI/ResultPrinter.php">
    <violation beginline="67"
               endline="224"
               rule="TooManyMethods"
               ruleset="Code Size Rules"
               package="PHP_Depend\DbusUI"
               class="PHP_Depend_DbusUI_ResultPrinter"
               priority="3">
      This class has too many methods, consider refactoring it.
    </violation>
  </file>
</pmd>
```

It seems it is a phpmd report file ran at 2009-12-19T22:17:18+01:00 with a single violation. As I use below example for my ruleset (which is PMD compatible):

```xml
<?xml version="1.0" ?>
<ruleset name="My rule set"
         xmlns="http://pmd.sf.net/ruleset/1.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd"
         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">

  <description>Check the code base for mess</description>

  <rule ref="rulesets/codesize.xml/CyclomaticComplexity"/>

  <exclude-pattern>*/*.config.php</exclude-pattern>
  <exclude-pattern>module/SomeDisabledModule</exclude-pattern>
</ruleset>
```

Or am I mistaken ?

**Comment by kylekatarnls** [comment]
Hi, this XML is a report, not a ruleset. Examples of ruleset files are here:
https://phpmd.org/documentation/creating-a-ruleset.html

**Comment by DannyvdSluijs** [comment]
@kylekatarnls that is exactly the issue as the line above reads:
> While the rulesets/codesize.xml ruleset file could look like this:

So either the line above is wrong and needs correction or the example needs replacing. In now see my report wasn't to clear on the details.
An alternative description could be
> While the output could look like this:

**Comment by kylekatarnls** [comment]
Thanks.
> While the rulesets/codesize.xml ruleset file could look like this:

rulesets/codesize.xml is an internal ruleset, you don't need to create this file to use it.

For example you can create a ruleset.xml file as below:
```xml
<?xml version="1.0"?>
<ruleset name="My first PHPMD rule set"
         xmlns="http://pmd.sf.net/ruleset/1.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
                     http://pmd.sf.net/ruleset_xml_schema.xsd"
         xsi:noNamespaceSchemaLocation="
                     http://pmd.sf.net/ruleset_xml_schema.xsd">
    <description>
        My custom rule set that checks my code...
    </description>

    <rule ref="rulesets/codesize.xml" />
    <rule ref="rulesets/cleancode.xml" />
    <rule ref="rulesets/controversial.xml" />
    <rule ref="rulesets/design.xml" />
    <rule ref="rulesets/naming.xml" />
    <rule ref="rulesets/unusedcode.xml" />
</ruleset>
```

Then if you have an `src` directory and want to get a text output, you can enter in your CLI:
```shell
phpmd src text ruleset.xml
```

I agree the doc is not clear enough. The documentation can be modified here:
https://github.com/phpmd/phpmd/blob/master/src/site/rst/documentation/index.rst

Your help is welcome.

**Comment by aarushlohit** [comment]
can i work???

**Comment by AJenbo** [comment]
@aarushlohit yes, but that this change in to consideration: https://github.com/phpmd/phpmd/pull/1298
